src/media/js/addon/actions/submit.js

```diff
@@ -131,7 +131,7 @@ export function create() {
       .then(res => {
         dispatch(submitOk(res.body));
       }, err => {
-        dispatch(submitError(err.response.body));
+        dispatch(validationFail(JSON.parse(err.response.text).detail));
```

Comment:
This is because the creation endpoint can also return a 400 error. Especially important for versions, since the version number conflicts checks can not be done earlier.